src/views/search/components/SaveSearchModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC, useMemo, useState } from 'react';
+
+import { ModalComponentProps } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Button,
+  Form,
+  FormGroup,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  TextInput,
+} from '@patternfly/react-core';
+
+type SaveSearchInputs = {
+  description: string;
+  name: string;
+};
+
+type SaveSearchModalProps = Pick<ModalComponentProps, 'isOpen' | 'onClose'> & {
+  onSubmit: (inputs: SaveSearchInputs) => void;
+};
+
+const SaveSearchModal: FC<SaveSearchModalProps> = ({ isOpen, onClose, onSubmit }) => {
+  const { t } = useKubevirtTranslation();
+
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  const isEmptyForm = useMemo<boolean>(
+    () => name === '' && description === '',
+    [name, description],
+  );
+
+  const submitForm = () => {
+    onSubmit({
+      description,
+      name,
+    });
+  };
+
+  return (
+    <Modal
+      className="ocs-modal co-catalog-page__overlay"
+      isOpen={isOpen}
+      onClose={onClose}
+      position="top"
+      variant="medium"
+    >
+      <ModalHeader title={t('Save search')} />
+      <ModalBody>
+        <Form>
+          <FormGroup label={t('Name')}>
+            <TextInput onChange={(_, value) => setName(value)} type="text" value={name} />
```

Comment:
Please, add some data-test-id attribute to the input. This will make testing much easier